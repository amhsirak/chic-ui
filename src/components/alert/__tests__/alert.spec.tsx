import Alert from '..';
import { render } from '../../../test/utils';

/* eslint-disable */
describe('Alert Component', () => {
  it('Renders Correctly', () => {
    const AlertGroup = render(
      <>
        <Alert title={'Title'} message={'message'}></Alert>
        <Alert type="secondary" title={'Title'} message={'message'}></Alert>
        <Alert type="warning" title={'Title'} message={'message'}></Alert>
        <Alert type="success" title={'Title'} message={'message'}></Alert>
        <Alert type="danger" title={'Title'} message={'message'}></Alert>
        <Alert type="info" title={'Title'} message={'message'}></Alert>
        <Alert type="light" title={'Title'} message={'message'}></Alert>
      </>
    );
    expect(AlertGroup).toMatchSnapshot();
  });
});
