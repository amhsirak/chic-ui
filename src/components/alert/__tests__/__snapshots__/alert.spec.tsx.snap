// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert Component Renders Correctly 1`] = `
Array [
  <div
    className="sc-bdfBQB lgkczG"
  >
    <div
      className="sc-dlfnuX bZAMYF"
    >
      Title
    </div>
    <div
      className="sc-hKgJUU lcNSlk"
    >
      message
    </div>
  </div>,
  <div
    className="sc-bdfBQB hEVYZg"
  >
    <div
      className="sc-dlfnuX bZAMYF"
    >
      Title
    </div>
    <div
      className="sc-hKgJUU lcNSlk"
    >
      message
    </div>
  </div>,
  <div
    className="sc-bdfBQB lkZOda"
  >
    <div
      className="sc-dlfnuX bZAMYF"
    >
      Title
    </div>
    <div
      className="sc-hKgJUU lcNSlk"
    >
      message
    </div>
  </div>,
  <div
    className="sc-bdfBQB kjEABS"
  >
    <div
      className="sc-dlfnuX bZAMYF"
    >
      Title
    </div>
    <div
      className="sc-hKgJUU lcNSlk"
    >
      message
    </div>
  </div>,
  <div
    className="sc-bdfBQB ga-DLbq"
  >
    <div
      className="sc-dlfnuX bZAMYF"
    >
      Title
    </div>
    <div
      className="sc-hKgJUU lcNSlk"
    >
      message
    </div>
  </div>,
  <div
    className="sc-bdfBQB cpednK"
  >
    <div
      className="sc-dlfnuX bZAMYF"
    >
      Title
    </div>
    <div
      className="sc-hKgJUU lcNSlk"
    >
      message
    </div>
  </div>,
  <div
    className="sc-bdfBQB jaKRqu"
  >
    <div
      className="sc-dlfnuX bZAMYF"
    >
      Title
    </div>
    <div
      className="sc-hKgJUU lcNSlk"
    >
      message
    </div>
  </div>,
]
`;
